modules/using-cli.adoc

```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
 [id="check-policy-compliance_{context}"]
 == Checking policy compliance
 
+You can use the `roxctl` CLI to check deployment YAML files and images for policy compliance.
+
+[discrete]
+=== Configuring output format
+When you check policy compliance by using the `deployment check`, `image check`, or `image scan` commands, you can specify the output format by using the `-o` option. This option determins how the output of a command is displayed in the terminal.
+
+You can change the output format by adding the `-o` option to the command and specifying the format as either `json`, `table`, `csv`, or `junit`.
+
+For examle, the following command checks a deployment and then displays the result in `csv` format:
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" \
+  deployment check -file =<yaml_filename> \
+  -o csv
+----
+
+[NOTE]
+====
+When you do not specify the `-o` option for the output format:
+
+* The default output format for the `deployment check` and the `image check` commands is `table`.
+* The default output format for the `image scan` command is `json`. This is the old JSON format output for compatibility with older versions of the CLI. To get the output in the new JSON format, specify the option with format, as `-o json`.
+====
+
+Depending on the format you select, there are different options available to configure the output. The following tabele lists the options for each format.
+
+[%header,cols="1,1,2"]
```

Comment:
This is a complex table with a fair amount of duplication. The complexity isn't the best from an accessibility standpoint. Consider reworking the way the information is presented in the table someday.